### PR TITLE
Import CKAN plugin

### LINKED ISSUE #188: Import CKAN plugin
Right now, the CKAN Trifid plugin is managed in a dedicated repository: https://github.com/zazuko/trifid-plugin-ckan.

To improve the maintenance of the plugin, the best would be to import it in the Trifid repository.

### PR DESCRIPTION
**Pull request by ludovicm67**
Closes #188.

### CODE REVIEW

File: .changeset/quick-bobcats-judge.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+---
+"@zazuko/trifid-plugin-ckan": minor
+---
+
+Use Zazuko libraries
```
Comment (tpluscode) [inline .changeset/quick-bobcats-judge.md:5]: Minor: maybe make it more specific?

File: packages/ckan/.editorconfig [hunk 2]
```diff
@@ -0,0 +1,5 @@
+[*.{js,jsx,ts,tsx,vue}]
+indent_style = space
+indent_size = 2
+trim_trailing_whitespace = true
+insert_final_newline = true
```
Comment (tpluscode) [inline packages/ckan/.editorconfig:5]: Looks like a sane config to put in root

File: packages/ckan/.gitignore [hunk 3]
```diff
@@ -0,0 +1,9 @@
+.DS_Store
+node_modules
+/dist
+.env
+
+# Log files
+npm-debug.log*
+yarn-debug.log*
+yarn-error.log*
```
Comment (tpluscode) [inline packages/ckan/.gitignore:9]: Ah, probably leftovers from old repo?

File: packages/ckan/src/namespace.js [hunk 4]
```diff
@@ -0,0 +1,8 @@
+import _rdf from '@zazuko/env'
+import prefixes from '@zazuko/prefixes'
+
+export const dcat = _rdf.namespace(prefixes.dcat)
+export const dcterms = _rdf.namespace(prefixes.dcterms)
+export const rdf = _rdf.namespace(prefixes.rdf)
+export const schema = _rdf.namespace(prefixes.schema)
+export const skos = _rdf.namespace(prefixes.skos)
```
Comment (tpluscode) [inline packages/ckan/src/namespace.js:8]: Zazuko env has all that. Might even remove the need for dependency on `@zazuko/prefixes`?

```suggestion
import rdf from '@zazuko/env'

const { dcat, dcterms, rdf, schema, skos } = rdf.ns

export { dcat, dcterms, rdf, schema, skos }
```

**Review by tpluscode — CHANGES_REQUESTED** [review]

File: packages/ckan/src/namespace.js (hunk 4 above)
Comment (ludovicm67) [reply packages/ckan/src/namespace.js]: I can't remove the dependency for `@zazuko/prefixes` as it's used in `src/xml.js`:

```js
import prefixes, { shrink } from '@zazuko/prefixes'
```

I also needed to rename `rdf` to `_rdf` back, as it's already in the export.

File: packages/ckan/.editorconfig (hunk 2 above)
Comment (ludovicm67) [reply packages/ckan/.editorconfig:5]: There is already one at the root, so I will remove this one.

File: packages/ckan/.gitignore (hunk 3 above)
Comment (ludovicm67) [reply packages/ckan/.gitignore:9]: I will remove entries listed in the root one, but keep specific ones for this plugin.

**Review by tpluscode — APPROVED** [review]

**Pull request merged**